src/main/java/seedu/address/logic/commands/HistoryCommand.java

```diff
@@ -15,7 +15,7 @@
 public class HistoryCommand extends Command {
 
     public static final String COMMAND_WORD = "history";
-    public static final String COMMAND_ALIAS = "i";
+    public static final String COMMAND_ALIAS = "his";
```

Comment:
UG needs to be updated to reflect this change

Reply:
updated.